etc/ProdOfflineConfiguration.py

```diff
@@ -1508,14 +1559,352 @@
                disk_node="T0_CH_CERN_Disk",
                dataset_lifetime=15*24*3600,
                scenario=ppScenario)
+#####################
+### HI TESTS 2024 ###
+#####################
+
+DATASETS = ["HIHcalNZS"]
+HIStreams += ["PhysicsHICommissioning"]
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               raw_to_disk=False,
+               aod_to_disk=False,
+               write_nanoaod=False,
+               write_dqm=True,
+               tape_node="T0_CH_CERN_MSS",
+               disk_node="T2_US_Vanderbilt",
+               alca_producers=["HcalCalMinBias"],
+               dqm_sequences=["@common", "@L1TMon", "@hcal"],
+               scenario=HIhcalnzsScenario)
+
+DATASETS = ["HIHLTPhysics"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               raw_to_disk=False,
+               aod_to_disk=False,
+               write_nanoaod=False,
+               write_dqm=True,
+               tape_node="T0_CH_CERN_MSS",
+               disk_node="T2_US_Vanderbilt",
+               alca_producers=["TkAlMinBias"],
+               dqm_sequences=["@common"],
+               scenario=hiScenario)
+
+DATASETS = ["HIOnlineMonitor", "HITrackerNZS"]
+HIStreams += ["DQM", "PhysicsHITrackerNZS"]
```

Comment:
```suggestion
HIStreams += ["HIDQM", "PhysicsHITrackerNZS"]
```
Starting 2024, most of the DQM streams in the PbPb menu will be named `HIDQM*`, to distinguish them more clearly from the pp menu (https://github.com/cms-sw/cmssw/pull/43645). I was not aware that HLT also needed to communicate to T0 the names of streams, in addition to the names of the Primary Datasets.

Reply:
This is not tipically the case. Our current setup require the use of stream names to divide the work into several agents. We are working on changing this so we can rely on PDs in the future.